repository_service_tuf/cli/key/generate_key.py

```diff
@@ -62,23 +62,16 @@ def generate() -> None:
 
     password = _verify_password(filename)
 
-    match key_type:
-        case KeyType.KEY_TYPE_ED25519.value:
-            _generate_and_write_ed25519_keypair(
-                password=password, filepath=filename
-            )
-
-        case KeyType.KEY_TYPE_ECDSA.value:
-            _generate_and_write_ecdsa_keypair(
-                password=password, filepath=filename
-            )
-
-        case KeyType.KEY_TYPE_RSA.value:
-            _generate_and_write_rsa_keypair(
-                password=password, filepath=filename
-            )
-
-        case _:  # pragma: no cover
-            # Current click configuration will never trigger this case, adding
-            # this as a fail-safe if we add new key-types
-            raise ValueError(f"Key type `{key_type}` is not supported!")
+    if key_type == KeyType.KEY_TYPE_ED25519.value:
+        _generate_and_write_ed25519_keypair(
+            password=password, filepath=filename
+        )
+    elif key_type == KeyType.KEY_TYPE_ECDSA.value:
+        _generate_and_write_ecdsa_keypair(password=password, filepath=filename)
+    elif key_type == KeyType.KEY_TYPE_RSA.value:
+        _generate_and_write_rsa_keypair(password=password, filepath=filename)
+    else:
+        # pragma: no cover
```

Comment:
Move the pragma here to avoid the coverage complain.
```suggestion
    else:  # pragma: no cover
        
```